src/components/Maps/AK_Fires.vue

```diff
@@ -72,13 +78,259 @@ export default {
+            return this.fireLayerGroup
+          },
+          'secondObject': () => { return this.secondFireLayerGroup },
+          'local': true,
+          'legend': false,
+          'abstract': '<img src="images/legend3.svg"/><p>This layer shows fires that occurred or are actively burning this year.</p><p>We update our map each hour from the source data available at the <a href="https://fire.ak.blm.gov" target="_blank" rel="externa">AICC</a> web site.</p><p><em>Where do most fires occur?  Where do most of the large fires occur?</em></p>'
+        }
+      ],
+      // Will initialize these in the created() method
+      activeFireIcon: undefined,
+      inactiveFireIcon: undefined,
+      firePolygons: undefined,
+      fireMarkers: undefined,
+      secondFirePolygons: undefined,
+      secondFireMarkers: undefined,
+      fireLayerGroup: undefined,
+      secondFireLayerGroup: undefined
+    }
+  },
+  created () {
+    let FireIcon = this.$L.Icon.extend({
+      options: {
+        iconUrl: '/static/active_fire.png',
+        iconSize: [30, 35],
+        shadowSize: [0, 0], // no shadow!
+        iconAnchor: [16, 34], // point of the icon which will correspond to marker's location
+        shadowAnchor: [0, 0],  // the same for the shadow
+        popupAnchor: [0, 0] // point from which the popup should open relative to the iconAnchor
+      }
+    })
+
+    this.activeFireIcon = new FireIcon()
+    this.inactiveFireIcon = new FireIcon({
+      iconUrl: '/static/inactive_fire.png'
+    })
+
+    // This will be the container for the fire markers & popups.
+    this.fireLayerGroup = this.$L.layerGroup()
+    this.secondFireLayerGroup = this.$L.layerGroup()
+
+    this.fetchFireData()
+  },
+  methods: {
+    getLocalLayers (layer) {
+      return {
+        first: this.fireLayerGroup,
+        second: this.secondFireLayerGroup
+      }
+    },
+    fetchFireData () {
+      return new Promise((resolve, reject) => {
+        if (this.firePolygons == null) {
+          this.$axios.get(window.fireFeaturesUrl)
+            .then(res => {
+              if (res) {
+                this.firePolygons = this.getGeoJsonLayer(res.data)
+                this.fireMarkers = this.getFireMarkers(res.data)
+                this.secondFirePolygons = this.getGeoJsonLayer(res.data)
+                this.secondFireMarkers = this.getFireMarkers(res.data)
+
+                // Add layers to the LayerGroup we're using here.
+                this.fireLayerGroup
+                  .addLayer(this.firePolygons)
+                  .addLayer(this.fireMarkers)
+                this.secondFireLayerGroup
+                  .addLayer(this.secondFirePolygons)
+                  .addLayer(this.secondFireMarkers)
+                resolve()
+              }
+            },
+            err => {
+              // TODO: handle this error with a popup warning
+              this.fireInfoPopup = false
+              console.log(err)
```

Comment:
Should we use `console.error()` instead for errors?

Reply:
Fixed, and also the assignment to `fireInfoPopup` was completely wrong.  Removed.